test/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <h1>Hello World!</h1>
+  </body>
+</html>
```

Comment:
This is where we have the problem of random characters popping up at the end; I did not find a solution to fix that yet.